src/test/java/timetables_sort_use_case/application_business/TestAllTimetablesInputBoundary.java

```diff
@@ -0,0 +1,14 @@
+package timetables_sort_use_case.application_business;
+
+public class TestAllTimetablesInputBoundary implements AllTimetablesInputBoundary {
+    private int j;
```

Comment:
Anonymous class in test file? (Applies to this and other test files). 👀
Having a lot of mocked up public classes might clutter things.

Reply:
same problem as mentioned in testalltimetablesview